### PR TITLE
feat: show merging user instead of OP

### PR DESCRIPTION
**Pull request by danieldietzler**

### CODE REVIEW

File: src/services/webhook.service.ts [hunk 1]
```diff
@@ -201,12 +202,13 @@ export class WebhookService {
   }
 
   private getEmbed(action: string, repositoryName: string, title: string, event: BaseEvent) {
+    const user = event.merged_by ? event.merged_by : event.user;
```
Comment (bo0tzz) [inline src/services/webhook.service.ts:205]:
```suggestion
    const user = event.merged_by || event.user;
```

I think this syntax is a thing right?

**Review by bo0tzz — APPROVED** [review]

File: src/services/webhook.service.ts (hunk 1 above)
Comment (danieldietzler) [reply src/services/webhook.service.ts:205]: Oh yeah woops, thanks!

**Review by jrasm91 — APPROVED** [review]

File: src/services/webhook.service.ts [hunk 2]
```diff
@@ -200,13 +200,13 @@ export class WebhookService {
     });
   }
 
-  private getEmbed(action: string, repositoryName: string, title: string, event: BaseEvent) {
+  private getEmbed(action: string, repositoryName: string, title: string, user: User, event: BaseEvent) {
```
Comment (jrasm91) [inline src/services/webhook.service.ts:203]: Maybe you should use named parameters 😁

Comment (danieldietzler) [reply src/services/webhook.service.ts:203]: Yeah I was thinking the same, it somewhat escalated 😅

**Pull request merged**